docs/_index.md

```diff
@@ -105,6 +105,9 @@ Viam supports a wide variety of systems, including:
   <div class="prev" style="display: block">‹</div>
   <div class="next" style="display: block">›</div>
 </div>
+<br>
+
+Explore the following resources to jump right in:
```

Comment:
Oooh let's not use the word "resources" here since it's got another meaning. How about
```suggestion
Get started with Viam:
```
I don't love that either but it's less potentially confusing? Open to other things for sure

Reply:
Oop! You're definitely correct! Will fixup -- thanks!